src/Actions/Actions/SelectAction.cs

```diff
@@ -214,36 +218,11 @@ public bool IsPaused
         /// <param name="eId"></param>
         public void SetCandidateElement(Guid ecId, int eId)
         {
-            var el = DataManager.GetDefaultInstance().GetA11yElement(ecId, eId).CloneForSelection();
+            var el = DataManager.GetA11yElement(ecId, eId).CloneForSelection();
 
             SetCandidateElement(el);
         }
 
-        /// <summary>
-        /// Set candidate element by process Id
-        /// </summary>
-        /// <param name="pid">process Id</param>
-        public void SetCandidateElementFromProcessId(int pId)
-        {
-            var handle = System.Diagnostics.Process.GetProcessById(pId).Handle;
-
-            SetCandidateElementFromHandle(handle);
-        }
-
-        /// <summary>
-        /// Set candidate element by handle.
-        /// </summary>
-        /// <param name="handle"></param>
-        public void SetCandidateElementFromHandle(IntPtr handle)
-        {
-            lock (_elementContextLock)
-            {
-                var element = A11yAutomation.ElementFromHandle(handle);
-                this.CandidateEC?.Dispose();
-                this.CandidateEC = new ElementContext(element);
-            }
-        }
-
```

Comment:
These methods are never called by AIWin and added extra complexity to properly handle with separate data, so it seemed better to remove them than to add extra complexity for stuff that we don't use